Tests/UnitTests/PatternsTest/SimplifyTest.cs

```diff
@@ -88,5 +89,18 @@ void AssertSimplify(Entity original, Entity simplified, int? level = null)
         [TestMethod] public void NaNP1() => AssertSimplify(nan + 1, nan);
         [TestMethod] public void NaN0() => AssertSimplify(nan * 0, nan);
         [TestMethod] public void NaNPow0() => AssertSimplify(MathS.Pow(nan, 0), nan);
+        [TestMethod] public void Derive1() => AssertSimplify(MathS.Derivative("x + 2", "x"), 1);
+        [TestMethod] public void Derive2() => AssertSimplify(MathS.Derivative("7x2 - x + 2", "x", 2), 14);
+        [TestMethod] public void Integral1() => AssertSimplify(MathS.Integral("x + y", "x", 0), "x + y");
+        [TestMethod] public void Divide1() => AssertSimplify("(x2 + 2 x y + y2) / (x + y)", "x + y");
+        [TestMethod] public void Divide2() => AssertSimplify("(x3 + 3 x 2 y + 3 x y 2 + y3) / (x + y)", "x2 + 2 x y + y2".Simplify());
+        [TestMethod] public void Divide3() => AssertSimplify("(x2 + 2 x y + y2 + 1) / (x + y)", "x + 1 / (x + y) + y".Simplify());
```

Comment:
`Simplify` being called on both sides does not ensure that the simplified result is acceptable - we should maintain an expected result that does not change with the implementation and update it when needed. Test should be more restrictive to actually catch bugs.

Reply:
@WhiteBlackGoose ?